features/facebookImport/src/components/postReactionTypesMiniStory/postReactionTypesMiniStory.jsx

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from "react";
+import { PolyChart } from "@polypoly-eu/poly-look";
+
+import InfoButton from "../buttons/infoButton/infoButton.jsx";
+
+import i18n from "../../i18n";
+
+import likeIcon from "../../static/images/reactions-ministory/like.svg";
```

Comment:
Please ensure that these icons fall under the license that's in the 3rd-party-license directory or add their license to that one.
Additionally, there's now a centralized directory for assets. Could you please consider placing every asset of this kind there?

Reply:
The icons are selfmade by Nic so licensing shouldn't be a problem. Will take a look at it!

Reply:
No worries about the 3rd party licenses then, but we still have to include the license for these somewhere.

Reply:
I believe so far we don't include anything for our own images - by default they're just under our copyright, which seems fine. if I'm right and we don't have an explicit license for our other SVGs, it's a topic unrelated to this PR in any case.

Reply:
By default they are under the repo LICENSE, which is GPL, not a great license for content. If we want to go with that, it's OK, because it's code at the end of the day (in this case). We might want to consider CC-BY-SA instead and include it in the specific directory, however.